src/libs/ComposerUtils/updateNumberOfLines/index.native.ts

```diff
@@ -8,7 +8,7 @@ import UpdateNumberOfLines from './types';
  * divide by line height to get the total number of rows for the textarea.
  */
 const updateNumberOfLines: UpdateNumberOfLines = (props, event) => {
-    const lineHeight = styles.textInputCompose.lineHeight;
+    const lineHeight = styles.textInputCompose.lineHeight ?? 0;
```

Comment:
why is change is necessary?

Reply:
did you change a logic or did TypeScript surfaces that lineHeight might be undefined?

Reply:
> or did TypeScript surfaces that lineHeight might be undefined

Correct 
<img width="512" alt="image" src="https://github.com/Expensify/App/assets/25230417/180c1e6f-81ca-49d7-986a-deaf66a9ff4c">